File: 351003/Mashevskiy/Lab1/src/main/java/by/tracker/rest_api/dto/TweetResponseTo.java
package by.tracker.rest_api.dto;

import lombok.Data;
import com.fasterxml.jackson.annotation.JsonProperty;
import java.time.LocalDateTime;

@Data
public class TweetResponseTo {
    private Long id;

    @JsonProperty("creatorId")  // может и не нужно, но для единообразия
    private Long creatorId;

    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}

File: 351002/Abrashin/publisher/src/main/kotlin/com/example/distcomp/dto/response/TweetResponseTo.kt
package com.example.distcomp.dto.response

import com.fasterxml.jackson.annotation.JsonRootName
import java.time.LocalDateTime

@JsonRootName("tweet")
data class TweetResponseTo(
    var id: Long? = null,
    var creatorId: Long? = null,
    var title: String? = null,
    var content: String? = null,
    var created: LocalDateTime? = null,
    var modified: LocalDateTime? = null,
    var stickers: List<StickerResponseTo> = emptyList()
)
